sections/4-EDI.md

```diff
@@ -1 +1,11 @@
-A maximum 1-page supplement that addresses EDI considerations and a plan for authorship for any manuscripts that are generated from the working group. In this supplement, please clearly identify the steps that will be taken to ensure broad representation and participation by diverse groups (e.g. include a plan for development of a code of conduct). Authorship considerations should explicitly state how authorship on manuscripts will be determined.
\ No newline at end of file
+*A maximum 1-page supplement that addresses EDI considerations and a plan for authorship for any manuscripts that are generated from the working group. In this supplement, please clearly identify the steps that will be taken to ensure broad representation and participation by diverse groups (e.g. include a plan for development of a code of conduct). Authorship considerations should explicitly state how authorship on manuscripts will be determined.*
+
+We recognize that equity, diversity, and inclusion (EDI) principles are critical to the mitigation of systemic barriers faced by marginalized populations in academia. In order to foster an inclusive and benevolent environment within our working group, all five organizers will complete an EDI training prior to the meetings, in addition to workshops focusing on the facilitation of group discussions. Moreover, we will collaboratively write a code of conduct with clear reporting mechanisms in case of misconduct. Our code of conduct will be open to suggestions by all working group participants, who will need to sign it before the first meeting. 
+
+Our list of participants is culturally diverse and respects gender parity. We mostly targeted students and early-career scientists who would benefit the most from a participation in this data-driven working group. We also invited researchers from different universities across Canada, making sure that their domains of expertise complement each other. With the aim of alleviating past and present inequalities experienced by marginalized groups and to encourage their participation in the working group, we will provide more funding to those who self-identified as being part of a marginalized community and provide a childcare service for mothers who would need it. Finally, participants who are unable to travel to Saskatchewan could attend the working group remotely if desired. 
```

Comment:
I like the idea for providing funding for those who self-identify as part of a marginalized community, but we haven't incorporated a way for participants to self-identify yet. Perhaps a pre-session survey/questionnaire could be incorporated into the pre-session activities (and could be mentioned in our work plan as well)